induction-task-do-you-know-this-is-a-test-rule/rule.md

```diff
@@ -0,0 +1,71 @@
+---
+type: rule
+archivedreason:
+title: {Induction task - Do you know this is a test rule?}
+guid: {b8d2dcb2-2123-4292-aed3-5471290aedce}
+uri: {induction-task}
+created: {2022-12-06T11:04:00.000Z}
+authors: [Jimmy Chen]
```

Comment:
Follow https://github.com/SSWConsulting/SSW.Rules.Content/wiki/How-to-Add-an-Acknowledgment